docs/dev/agent_build.md

```diff
@@ -52,6 +52,30 @@ Also note that the trace agent needs to be built and run separately. For more in
 We use `pkg-config` to make compilers and linkers aware of Python. The required .pc files are
 provided automatically when building python through omnibus.
 
+As an option, the Agent can combine multiple functionalities into a single binary to reduce
+the space used on disk. The `DD_BUNDLED_AGENT` environment variable is used to select
+which functionality to enable. For instance, if set to `process-agent`, it will act as the process Agent.
+If the environment variable is not defined, the process name is used as a fallback.
+As the last resort meaning, the executable will behave as the 'main' Agent.
+
+Different combinations can be obtained through the usage of build tags. As an example,
+building the Agent with the `bundle_process_agent` and `bundle_security_agent` will produce
+a binary that has the process Agent and security Agent capabilities.
+
+The `--bundle` argument can be used to override the default set of functionalities bundled
+into the Agent binary. For instance, to override the defaults and bundle only the process and
+and the security Agents:
+
+```
+deva agent.build --bundle process-agent --bundle security-agent
+```
+
+To disable bundling entirely:
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Sounds confusing that this doc says that the agent can combine multiple functionalities as an option, and then here it says that to disable bundling you have to pass `--bundle agent`. What is the default?

Reply:
If you don't put anything it's equivalent to `--bundle agent`